client/app/vps/upgrade/vps-upgrade.html

```diff
@@ -0,0 +1,114 @@
+<section class="container-fluid cui-page__content cui-page__content-flex">
+
+    <div class="row d-lg-flex">
+        <div class="col-xm-12">
+
+            <cui-wizard-form data-form-title="{{ 'vps_dashboard_upgrade_vps' | translate }}"
+                             data-form-loaded="!$ctrl.loaders.init"
+                             data-form-on-cancel="$ctrl.resetAction()"
+                             data-form-on-complete="$ctrl.displayBc()">
+
+                    <cui-wizard-step data-step-title="{{ 'vps_configuration_upgradevps_step1_question' | translate }}"
+                                     data-step-on-init="$ctrl.loadUpgradesList()"
+                                     data-step-loaded="!$ctrl.loaders.step1"
+                                     data-step-completed="$ctrl.selectedModel.model"
+                                     data-step-on-complete="$ctrl.loadNextStep(2)">
+                        <div class="oui-radio"
```

Comment:
Use ``<oui-radio-group> `` instead, see http://master.ui-kit.ovh/#!/oui-angular/radio-group

Reply:
The radio buttons are too complex to be used in a `<oui-radio-group>` : they contain almost a table row in it

Reply:
what do you mean by that?  I used radio-group multiple times and ti works well...

Reply:
In the sample presented by ui-kit, the `label` attribute is just a simple text, whereas here we need to display a complex DOM element inside this attribute